tools/file_packager.py

```diff
@@ -282,8 +282,7 @@ def main():
   # standalone calls
   if not from_emcc:
     ret = '''
-  var Module = typeof %(EXPORT_NAME)s !== 'undefined' ? %(EXPORT_NAME)s : {};
-  ''' % {"EXPORT_NAME": export_name}
+  var Module = typeof %(EXPORT_NAME)s !== 'undefined' ? %(EXPORT_NAME)s : {};\n''' % {"EXPORT_NAME": export_name}
```

Comment:
> The three spaces IIUC were there to avoid the """ being in column zero or the python code.

Can we just remove the three spaces then? Python accepts that IIANM.

Reply:
Yes, but the problem is that we end up with lots of stuff in column 1 of the python code.. which IMHO makes it harder to read in general.. and add a lot of extra lines to the lines don't do anything.

I've updated the this PR to be even more consistent and added quick test for trailing/extra whitespace in the output.

Reply:
sgtm, I'm mostly neutral here, so I don't want to block this. Good idea about the test.